src/sparseml/transformers/utils/helpers.py

```diff
@@ -224,3 +211,106 @@ def resolve_sequence_length(config: AutoConfig) -> int:
         "(inferred from HF transformers config) "
     )
     return sequence_length
+
+
+def resolve_recipe_application(
```

Comment:
Could we update the name to be `resolve_recipe`? As we are not really applying the recipe in this function, name is  sorta misleading?

Reply:
Good call!